### ISSUE TITLE
Passing !important as a function argument causes "check validity of the block" error

### ISSUE THREAD

**Issue by jeffkamo** [sasstools/sass-lint#810]
**What version of Sass Lint are you using?**
1.8.2

**What did you do?**
In my code, I had the following (which indeed [should work](http://www.sassmeister.com/gist/9d25b986c2003d52261b563cae3f6fa5)):

``` scss
@mixin test($border: 1px solid red) {
  border: $border;
}

.test1 {
  @include test;
}

.test2 {
  @include test(5px dashed blue);
}

.test3 {
  @include test(10px solid green !important);
}
```

Notice the third instance of `@include test()` has `!important` passed into it as part its argument. This is the cause of the error, as per below.

**What did you expect to happen?**
I expected that that sass-lint passes without issue. It appears to me that Sass is able to handle having `!important` be passed to a function as a parameter without issue.

**What actually happened?**
The linter errors out, giving a message of...

```
error  Please check validity of the block starting from line #x
```

...where `line #x` is where the `!important` was passed into the function as a parameter. The full error log is as follows:

```
$ npm run lint

> ...-scss@2.0.0 lint /Users/.../.../...

> sass-lint -c node_modules/.../css/.sass-lint.yml 'dist/**/*.scss' -v -i 'test/**/*.scss'


dist/utilities/color/_color.scss
  14:1  error  Please check validity of the block starting from line #14  Fatal

✖ 1 problem (1 error, 0 warnings)

/.../node_modules/sass-lint/index.js:294
    throw new Error(errorCount.count + ' errors were detected in \n- ' + errorCount.files.join('\n- '));
    ^

Error: 1 errors were detected in 
- dist/utilities/color/_color.scss
    at Function.sassLint.failOnError (/.../node_modules/sass-lint/index.js:294:11)
    at detectPattern (/.../node_modules/sass-lint/bin/sass-lint.js:33:10)
    at /.../node_modules/sass-lint/bin/sass-lint.js:98:5
    at Array.forEach (native)
    at Object.<anonymous> (/.../node_modules/sass-lint/bin/sass-lint.js:97:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)

npm ERR! Darwin 15.5.0
npm ERR! argv "/.../.nvm/versions/node/v4.4.0/bin/node" "/.../.nvm/versions/node/v4.4.0/bin/npm" "run" "lint"
npm ERR! node v4.4.0
npm ERR! npm  v2.14.20
npm ERR! code ELIFECYCLE
npm ERR! ...-scss@2.0.0 lint: `sass-lint -c node_modules/.../css/.sass-lint.yml 'dist/**/*.scss' -v -i 'test/**/*.scss'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ...-scss@2.0.0 lint script 'sass-lint -c node_modules/.../css/.sass-lint.yml 'dist/**/*.scss' -v -i 'test/**/*.scss''.
npm ERR! This is most likely a problem with the ...-scss package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sass-lint -c node_modules/.../css/.sass-lint.yml 'dist/**/*.scss' -v -i 'test/**/*.scss'
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ...-scss
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls ...-scss
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /.../npm-debug.log
```

**Comment by BPScott** [comment]
Sounds like a bug in the AST. Can you report it at https://github.com/tonyganch/gonzales-pe

**Comment by jeffkamo** [comment]
Thanks @BPScott, it looks like it's a known issue for them. Someone posted an issue about it [here](https://github.com/tonyganch/gonzales-pe/issues/168)!

**Comment by bgriffith** [comment]
This is fixed in the newly released version v1.11.0. Please let me know if you still have issues. Thanks!
